tests/console/validate_yast2_firstboot_configuration.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Test module to validate YaST Firstboot configuration settings are
+# applied to the SUT.
+# Maintainer: Oleksandr Orlov <oorlov@suse.de>
+
+package validate_yast2_firstboot_configuration;
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+
+sub pre_run_hook {
+    select_console 'root-console';
+}
+
+sub run {
+    assert_user_exist(get_var('YAST2_FIRSTBOOT_USERNAME'));
+}
+
+sub assert_user_exist {
```

Comment:
I would write the function before `run` to avoid new tester classic mistake to put sub without parentheses after they are used. In this case it's fine because you do `get_var` from outside using ().

Reply:
Done.